apollo-router/src/plugins/cache/cache_control.rs

```diff
@@ -178,6 +178,16 @@ impl CacheControl {
         let mut s = String::new();
         let mut prev = false;
         let now = now_epoch_seconds();
+        if self.no_store {
+            write!(&mut s, "no-store")?;
+            // Early return to avoid conflicts https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#preventing_storing
+            return Ok(s);
+        }
+        if self.no_cache {
+            write!(&mut s, "no_cache")?;
```

Comment:
```suggestion
            write!(&mut s, "no-cache")?;
```
I think it should have dash, or I missing something?

Reply:
Here it has dash https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#no-cache_2

Reply:
Oh wow ! Good catch !